Java Migration/TFS_2018Update2.md

```diff
@@ -0,0 +1,41 @@
+
+Below diagram explains the steps to migrate from Oracle JRE to Azul Zulu OpenJDK for TFS 2018 Update 2.
+
+![Java Migration flow](flow1.png)
+
+## Step 1: Pause Search indexing
+Go to https://github.com/Microsoft/Code-Search and find the right folder based on the TFS version you are using. For TFS 2018 Update 2, go to https://github.com/Microsoft/Code-Search/tree/master/TFS_2017Update2. To pause all indexing, execute the script PauseSearchIndexing.ps1 from Windows PowerShell with administrative privileges. You will be prompted to enter:
+
+* The SQL server instance name where the TFS configuration database resides.
+* The name of the TFS configuration database.
+
+## Step 2: Stop Elasticsearch Service
+Open Command Prompt as an administrator 
+
+If Elasticsearch is installed on the same server as TFS (local installation), use below command to locate ES folder. For remote search installations, locate the ES installation path and change the directory accordingly.
+### Change directory: 
+cd "C:\Program Files\Microsoft Team Foundation Server 15.0\Search\ES\elasticsearch-5.4.1\bin"
+### Stop the service:
+execute "elasticsearch-service.bat stop"
+
+## Step 3: Download and Install Azul Zulu Java 
+Download and install [OpenJDK 7u201](https://cdn.azul.com/zulu/bin/zulu7.25.0.5-jdk7.0.201-win_x64.msi)
+
+## Step 4: Update JAVA_HOME with Azul Zulu path
+![Update Java Home](java_home.png)
+
+## Step 5: Start Elasticsearch Service
+Open Command Prompt as an administrator 
+
+If Elasticsearch is installed on the same server as TFS (local installation), use below command to locate ES folder. For remote search installations, locate the ES installation path and change the directory accordingly.
+### Change directory: 
+cd "C:\Program Files\Microsoft Team Foundation Server 15.0\Search\ES\elasticsearch-5.4.1\bin"
```

Comment:
check once, i think it should be 16.0 not 15.0

Reply:
fixed it